target/src/package.json

```diff
@@ -1,10 +1,9 @@
 {
   "name": "antlr4ts",
-  "version": "0.4.0-dev",
+  "version": "0.4.1-alpha.0",
```

Comment:
I think it's OK now.   I'll try to finish up on my work-in-progress of the cleaner solution of copying from root.